### DISCUSSION TITLE
Welcome to Inployd Discussions!

### DISCUSSION THREAD

**Discussion by jakezam** [jakezam/crabapple-conglomerate#27] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 0NotApplicable0** [comment]
Hey everyone, I'm Nick! Jake and I started crunching ideas for this project a while back. Hopefully we can get a MVP done sometime soon. We don't plan on really bring this into production but more of a learning experiences. But hey, who knows, maybe we can make something out of this!

**Reply by faxonchris** [reply]
I think this project has a lot of potential as a FOSS alternative to many popular freelance sites
